Core/Core/Data/Repository/AuthRepository.swift

```diff
@@ -80,6 +81,18 @@ public class AuthRepository: AuthRepositoryProtocol {
         return user.domain
     }
 
+    public func SSOlogin(jwtToken: String) async throws -> User {
+        if appStorage.accessToken == nil ||
+            appStorage.refreshToken == nil {
+            appStorage.accessToken = jwtToken
+            appStorage.refreshToken = jwtToken
```

Comment:
Unfortunately, I cannot test this code, but usually, `accessToken` differs from the `refreshToken` for security reasons. 
Please check if this approach works and refresh the token.

Reply:
I agree, refreshToken isn't provided via the cookie so we should keep it null if possible.

Reply:
✅